script.js

```diff
@@ -1,10 +1,14 @@
 function shifumi() {
     const user_choice = ['rock', 'paper', 'scissors']
-    const choice = prompt(`Choose one of the following : ${user_choice.join(', ')}.`)
-    if (user_choice.includes(choice)) {
-        alert(`Congratulations, you chose ${choice}`)
-    } else {
-        alert(`you did not choose one of the ${user_choice.length} choices`)
-    }
-}
-shifumi();
\ No newline at end of file
+    let choice = prompt(`Choose one of the following : ${user_choice.join(', ')}.`)
+    if (!user_choice.includes(choice)) {
+        alert(`you did not choose one of the ${user_choice.length} choices`);
+        choice = prompt(`Choose one of the following : ${user_choice.join(', ')}, last chance!`)
```

Comment:
indeed, this is working. What if you fail 3,4,5,100 times in a row? How would you manage this situation? (we can have a quick call if you need some guidance)